pyfolio/tests/test_timeseries.py

```diff
@@ -311,12 +311,13 @@ class TestStats(TestCase):
         (simple_month_rets, utils.MONTHLY, 0.006885932704891129)
     ])
     def test_annual_ret(self, returns, period, expected):
-        self.assertEqual(
+        assert_almost_equal(
```

Comment:
The test test passes on master on my computer. The issue came up when switched over to qrisk.